.gitignore

```diff
@@ -0,0 +1,8 @@
+app/recipes/pico/*
+app/recipes/zymatic/*
+app/recipes/zseries/*
+
+app/sessions/brew/active/*
+app/sessions/brew/archive/*
+app/sessions/ferm/active/*
+app/sessions/ferm/archive/*
```

Comment:
I keep getting these as untracked changes thus explicitly ignoring them in git...